client/src/app-components/ticket-viewer.js

```diff
@@ -78,7 +81,21 @@ class TicketViewer extends React.Component {
                 </div>
                 {this.props.editable ? this.renderEditableHeaders() : this.renderHeaders()}
                 <div className="ticket-viewer__content">
-                    <TicketEvent type="COMMENT" author={ticket.author} content={this.props.userStaff ? MentionsParser.parse(ticket.content) : ticket.content} date={ticket.date} file={ticket.file}/>
+                    <TicketEvent
+                        loading={this.state.loading}
+                        type="COMMENT"
+                        author={ticket.author}
+                        content={this.props.userStaff ? MentionsParser.parse(ticket.content) : ticket.content}
+                        userStaff={this.props.userStaff}
+                        userId={this.props.userId}
+                        date={ticket.date}
+                        onEdit={this.onEdit.bind(this,0)}
+                        edited={ticket.edited}
+                        file={ticket.file}
+                        edit={this.state.edit && this.state.editId == 0}
```

Comment:
We should add a condition to ensure that logged `id` is the same as the `id` of the comment author